src/components/Leaderboard.js

```diff
@@ -0,0 +1,25 @@
+import styled from "styled-components";
+import LeaderboardBtn from "./LeaderboardBtn";
+
+const LeaderboardMolecule = styled.div`
+  /* background: black; */
+  margin: 0 auto 0 auto;
+  width: 700px;
+  display: block;
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  /* outline: 2px dotted black; */
+`;
+
+const Leaderboard = (props) => {
+  return (
+    <LeaderboardMolecule>
```

Comment:
Not sure if this needs to be called a molecule, maybe a naming convention would be something you could discuss with the team?